manticore/ethereum/__init__.py

```diff
@@ -672,6 +672,24 @@ def solidity_create_contract(self, source_code, owner, name=None, contract_name=
             return None
         return contract_account
 
+    def get_nonce(self, address):
+        # type forgiveness:
+        address = int(address)
+        # get all states containing this address:
+        states = tuple(state for state in self.running_states if address in state.platform)
```

Comment:
Note that this for read a state from storage and write them back in each iteration. So it is better if we can collapse this and the next for_all iteration thing